### PR TITLE
Add more scenarios to fundamentals

### PR DESCRIPTION
**Pull request by franciscoaguirre**
Added scenarios for:
- [x] Locations
- [x] Assets
- [x] Instructions
- [ ] Executor (open question)
- [ ] Pallet XCM

### CODE REVIEW

**Comment by franciscoaguirre** [comment]
@shawntabrizi, please let me know if this is the kind of contribution you want

**Comment by shawntabrizi** [comment]
Yes! 

For the location stuff we will need some scenarios since location for the same object will be different based on perspective.

File: src/fundamentals/xcm_executor.rs [hunk 1]
```diff
@@ -34,12 +36,34 @@ pub trait XcmConfig {
 }
```
Comment (franciscoaguirre) [inline src/fundamentals/xcm_executor.rs:36]: Any reason for exposing `AssetTransactor` and `Barrier` to them in these fundamentals?

I could only make it work by removing the `start` feature and using the `example` feature for using the available `AssetTransactor` there.
Do we want them to implement their own at this stage? If that's the case, I'll probably just create a new simpler trait with only `withdraw` and `deposit`.

I think a good aspect of this "fundamentals" section is we tackle things one at a time, so maybe not a good idea to put the `Barrier` here already. That could go in a separate lesson on setting up the barrier and only the barrier.

**Comment by franciscoaguirre** [comment]
@shawntabrizi I added more scenarios for locations, assets and instructions. Also left a comment about the executor exercise.

My idea is that all of this gets taught in a lecture first, they use the workshop as a way of actually writing the code and letting those concepts sink in.

Locations, assets, the most relevant instructions, the most relevant config items and versioning can be presented in the lecture and further expanded upon here.

I'll create a draft of the lectures.

File: src/fundamentals/xcm_executor.rs (hunk 1 above)
Comment (franciscoaguirre) [reply src/fundamentals/xcm_executor.rs:36]: Maybe they should just tackle this after the fundamentals, and we should stick to the simplest version of the executor here.

**Comment by franciscoaguirre** [comment]
Started work on the new slides: https://github.com/Polkadot-Blockchain-Academy/pba-content/pull/1001
Will keep on working on that PR during the rest of the week

**Review by shawntabrizi — APPROVED** [review]

File: src/fundamentals/xcm_executor.rs (hunk 1 above)
Comment (shawntabrizi) [reply src/fundamentals/xcm_executor.rs:36]: >  Do we want them to implement their own at this stage? If that's the case, I'll probably just create a new simpler trait with only withdraw and deposit.

Yes, i think something like this

**Pull request merged**